hadoop-mapreduce/src/test/java/org/apache/accumulo/hadoop/its/mapreduce/AccumuloFileOutputFormatIT.java

```diff
@@ -72,8 +73,8 @@
           .addOption("modulus", "3");
 
   @Override
-  protected int defaultTimeoutSeconds() {
-    return 4 * 60;
+  protected Duration defaultTimeoutDuration() {
```

Comment:
If the method is going to be renamed anyway, `defaultTimeout` is sufficient. The type doesn't need to be included. It was only ever included before because we needed to know that it was seconds, and not, for example, millis.

Reply:
Good idea. Addressed in [546a85d](https://github.com/apache/accumulo/pull/2580/commits/546a85dd508cfb0f2508d8372ed10df843bd6e52)